app/uk/gov/hmrc/securemessage/controllers/SecureMessageController.scala

```diff
@@ -38,7 +42,12 @@ class SecureMessageController @Inject()(
   def createConversation(client: String, conversationId: String): Action[JsValue] = Action.async(parse.json) {
     implicit request =>
       withJsonBody[ConversationRequest] { conversationRequest =>
-        secureMessageService.createConversation(conversationRequest, client, conversationId)
+        secureMessageService.createConversation(conversationRequest.asConversation(client, conversationId)).map {
+          case Right(_)                          => Created
+          case Left(ee: EmailException)          => Created(ee.message)
+          case Left(se: StoreException)          => Conflict(se.message)
+          case Left(sme: SecureMessageException) => Created(sme.message)
```

Comment:
what are other exception ? 
I guess here should be case Left(error: Throwable) =>
   logger.error(error.message, error) 
   INTERNAL_SEVER_ERROR(error.message)